src/main/resources/application.yml

```diff
@@ -70,9 +70,9 @@ prd:
 
 docmosis:
   service:
-    uri: ${PDF_SERVICE_BASEURL:https://docmosis.aat.platform.hmcts.net}
```

Comment:
would we be better updating the `PDF_SERVICE_BASEURL` (and the other values we are updating in this review) in our configuration source rather than changing the default value?